Actors/ActorCounter/Scripts/upgrade.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -x
+azure servicefabric application package copy CounterActorApplication fabric:ImageStore > /dev/null 2>&1
```

Comment:
yes, it should. we need to run like ``bash Scripts/install.sh`` to do it from terminal at the root/project level. That's how it works internally in eclipse plugin generated SF templates. I just followed the same structure here.